src/mad_spark_alt/core/conclusion_synthesizer.py

```
@@ -0,0 +1,304 @@
+        self,
+        problem_statement: str,
+        ideas_by_phase: Dict[str, List[GeneratedIdea]]
+    ) -> Conclusion:
+        """Synthesize conclusion using templates when LLM unavailable."""
+        
+        # Count total ideas
+        total_ideas = sum(len(ideas) for ideas in ideas_by_phase.values())
+        
+        # Extract key themes
+        all_ideas = []
+        for phase, ideas in ideas_by_phase.items():
```

Ruff (0.11.9)

Loop control variable `phase` not used within loop body

Rename unused `phase` to `_phase`

(B007)